elementary/website
# Use cache-busted versions of scripts on docs pages

In #3126 we added cache busting by appending checksums to our script and css filenames.

This works for most pages, but I've noticed some of the markdown pages (i.e. the installation guide) include scripts inside the markdown, so we need to add some code to replace these filenames from the manifest.

Nothing is broken on the server here currently as the only javascript file without the checksum is still present on the server, but this may break in the future if we don't do this.

To test:

* Run site locally, go to http://localhost:8000/docs/installation 
* See that the `installation.js` file cannot be loaded
* Run this branch, see that the correct script filename is substituted and loaded
